lib/Contentful.js

```diff
@@ -152,8 +152,13 @@ class Contentful {
         if (!(entry instanceof Object)) {
           return entry;
         }
+        
+        // Assets are not indexed
+        if (entry.sys.type === 'Asset') return m
```

Comment:
Why should assets not be indexed within the result? For instance I want all my image urls to show up here.